services/QuillLMS/app/controllers/ortto_integration/webhooks_controller.rb

```diff
@@ -3,35 +3,27 @@
 module OrttoIntegration
   class WebhooksController < ApplicationController
     protect_from_forgery except: :create
-    before_action :valid_params?, only: [:create]
+    before_action :authenticate
 
     class OrttoWebhookBadRequestException < StandardError; end
 
-    VALID_ACTIONS = [
-      SUBSCRIBE   = 'subscribe',
-      UNSUBSCRIBE = 'unsubscribe'
-    ]
-
     def create
-      unless valid_params?
+      email = params[:email]
+      user = User.find_by_email(email)
+
+      if email.nil? || user.nil?
         ErrorNotifier.report(
           OrttoWebhookBadRequestException.new("Bad request with params: #{params}")
         )
-        return head(:bad_request)
+        return head :accepted
```

Comment:
Using HTTP 202 here will allow the initial 'test connection', and helps developers disambiguate between the test request and a normal request to this API.